server/app/Module.scala

```diff
@@ -0,0 +1,9 @@
+import com.google.inject.AbstractModule
+import com.google.inject.name.Names.named
+
+import scala.util.Random.nextInt
+
+class Module extends AbstractModule {
+  override def configure() =
```

Comment:
I agree, but in case if `ImplemenetedBy` is in the same file as implemetation it is ok for me. You don't need to
>run over all classes in project

Don't you?